packages/api/src/hooks/useLandingCompany.ts

```diff
@@ -12,13 +12,11 @@ const useLandingCompany = () => {
 
     const modified_data = useMemo(() => {
         if (!data?.landing_company) return;
-        const { financial_company, gaming_company, virtual_company } = data.landing_company;
+        const { financial_company, virtual_company } = data.landing_company;
```

Comment:
Should we do this tho ? 😨

Reply:
I still return gaming company in `...data.landing_company`. Just not explicitly modified the data to return the shortcode of gaming_company. But, will we still be using it? Based on jy reviews

Reply:
i think its still using elsewhere, better to keep it there for the time being 🙏